supply/lib/supply/options.rb

```diff
@@ -210,6 +210,12 @@ def self.available_options
                                      description: "Whether to skip uploading SCREENSHOTS",
                                      type: Boolean,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :sync_image_upload,
+                                     env_name: "SUPPLY_SYNC_IMAGE_UPLOAD",
+                                     optional: true,
```

Comment:
Does it need to be optional since it's just a boolean that has a default value?

Reply:
Good point 👍